code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -377,12 +377,12 @@
 	for(var/mob/M in view(7, src))
 		if(M.stat != CONSCIOUS) //Check if it's conscious FIRST.
 			continue
-		else if(istype(M, childtype)) //Check for children SECOND.
+		else if(is_type_in_typecache(M, childtype)) //Check for children SECOND.
 			children++
 		else if(istype(M, animal_species))
 			if(M.ckey)
 				continue
-			else if(!istype(M, childtype) && M.gender == MALE) //Better safe than sorry ;_;
+			else if(!is_type_in_typecache(M, childtype) && M.gender == MALE) //Better safe than sorry ;_;
```

Comment:
can lose this.

Reply:
see above "there's a "better safe than sorry" comment so I assume there's a reason why there are two checks"